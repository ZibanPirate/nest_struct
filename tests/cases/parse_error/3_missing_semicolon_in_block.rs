#![allow(dead_code)]
#[macro_use]
extern crate nest_struct;

#[nest_struct]
struct NestedBlock {
    id: u32,
    nested_block: nest! {
        enum NestedEnum {
            First
            Last,
        },
    },
}

fn main() {}
