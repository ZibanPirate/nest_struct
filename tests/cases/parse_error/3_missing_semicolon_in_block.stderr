error: if nesting a struct: expected identifier, found keyword `enum`
 --> tests/cases/parse_error/3_missing_semicolon_in_block.rs:9:9
  |
9 |         enum NestedEnum {
  |         ^^^^

error: if nesting an enum: expected identifier, found keyword `enum`
 --> tests/cases/parse_error/3_missing_semicolon_in_block.rs:9:9
  |
9 |         enum NestedEnum {
  |         ^^^^

error: if nesting a block: expected `,`
  --> tests/cases/parse_error/3_missing_semicolon_in_block.rs:11:13
   |
11 |             Last,
   |             ^^^^
